OCPBUGS-19072#updating text (#74142)
openshift/openshift-docs: applications/deployments/route-based-deployment-strategies.adoc

The most common route-based strategy is to use a _blue-green deployment_. The new version (the green version) is brought up for testing and evaluation, while the users still use the stable version (the blue version). When ready, the users are switched to the green version. If a problem arises, you can switch back to the blue version.

> A common alternative strategy is to use _A/B versions_ that are both active at the same time. This allows some users to use _version_A_, and others could use _version_B_. This strategy could be used for experimenting with user interface changes or other features in order to get user feedback. It could also be used to verify proper operation in a production context where problems impact a limited number of users.

Reviewer: Optional suggestion to make this more active voice.

```suggestion
Alternatively, you can use an _A/B versions_ strategy in which both versions are active at the same time. With this strategy, some users could use _version_A_, and other users could use _version_B_. You could use this strategy to experiment with user interface changes or other features in order to get user feedback. You could also use it to verify proper operation in a production context where problems impact a limited number of users.
```

> Alternatively, you can use an _A/B versions_ strategy in which both versions are active at the same time. With this strategy, some users could use _version_A_, and other users could use _version_B_. You could use this strategy to experiment with user interface changes or other features in order to get user feedback. You could also use it to verify proper operation in a production context where problems impact a limited number of users.

Reviewer 2: ```suggestion
Alternatively, you can use an _A/B versions_ strategy in which both versions are active at the same time. With this strategy, some users could use _version_A_, and other users could use _version_B_. You could use this strategy to experiment with user interface changes or other features in order to get user feedback. You can also use it to verify proper operation in a production context where problems impact a limited number of users.
```

Reviewer 2: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c:~:text=file%20is%20corrupted%E2%80%9D).-,could%20verb,-Do%20not%20use

The most common route-based strategy is to use a _blue-green deployment_. The new version (the green version) is brought up for testing and evaluation, while the users still use the stable version (the blue version). When ready, the users are switched to the green version. If a problem arises, you can switch back to the blue version.

> One alternative strategy is to use _A/B versions_ that are both active at the same time and some users use one version, and some users use the other version. This can be used for experimenting with user interface changes and other features to get user feedback. It can also be used to verify proper operation in a production context where problems impact a limited number of users.

Reviewer 3: ```suggestion
One alternative strategy is to use _A/B versions_ that are both active at the same time.  This allows some users to use _version A_, and others could use _version B_. This strategy could be used for experimenting with user interface changes or other features in order to get user feedback. It could also be used to verify proper operation in a production context where problems impact a limited number of users.
```

Author: @mburke5678, Thanks Michael, I applied those changes.
